pkgs/by-name/hu/hurl/package.nix

```diff
@@ -39,6 +39,13 @@ rustPlatform.buildRustPackage rec {
 
   postInstall = ''
     installManPage docs/manual/hurl.1 docs/manual/hurlfmt.1
+    installShellCompletion --cmd hurl \
+      --bash completions/hurl.bash \
+      --zsh completions/_hurl \
+      --fish completions/hurl.fish
+
+    installShellCompletion --cmd hurlfmt \
```

Comment:
Not available for  the other shells?

Reply:
I dont think so https://github.com/Orange-OpenSource/hurl/tree/master/completions